db/rdbms/src/main/resources/db/vendor-properties/h2.properties

```diff
@@ -9,3 +9,4 @@
 paging.after=LIMIT #{page.size} OFFSET #{page.from}
 keysetPaging.limit=LIMIT #{page.size}
 variableValue.previewSize=8191
+escapeChar='\\'
```

Comment:
❓ Is this configurable, so customers can set a different escape char? Do we need to consider this on the application level?

Reply:
Right now it's not configurable. We could probably make it, although it is also DB specific, so we would need to add multiple properties.

I'll need test it with values containing backslashes, like `C:\Users` or similar. But I think in worst case we also need to escape these backslashes, too. I don't think it needs to be configurable though.

Reply:
So, this hardwires the properties for our connection to the specific database? If that's the case, then this can stay as-is. As soon as this is configurable by users, we need to consider this in the wildcard transformation code as well.

Reply:
Perfect 👍 Yeah, sorry, I didn't make that very specific in the beginning 🙈